assets/src/features/room-page/components/Navbar.tsx

```diff
@@ -4,17 +4,24 @@ import { useParams } from "react-router-dom";
 import Button from "../../shared/components/Button";
 import MembraneVideoroomLogo from "../../shared/components/MembraneVideoroomLogo";
 import PlainLink from "../../shared/components/PlainLink";
+import useToast from "../../shared/hooks/useToast";
 import ShareSquare from "../icons/ShareSquare";
 
 const Navbar: React.FC = () => {
   const match = useParams();
   const currentUrl = window.location.href;
   const roomId: string = match?.roomId || "";
+  const { addToast } = useToast();
 
   const copyLink = useCallback(async () => {
     await navigator.clipboard.writeText(currentUrl);
   }, [currentUrl]);
 
+  const onLinkCopy = useCallback(() => {
+    copyLink();
+    addToast({ id: "toast-link-copied", message: "Link copied to clipboard" });
+  }, [addToast, copyLink]);
```

Comment:
Since`coypLink` is used only in the `onLinkCopy` function it would be better to use it inside the `onLinkCopy` function